### PR TITLE
Parse Russian PubIDs

### LINKED ISSUE #8: Internationalized PubIDs: Russian
ISO provides PubIDs in Russian:

* ISO is ИСО
* ISO/IEC is ИСО/МЭК
* TR is ТО
* TS is ТС
* DIS is ПМС
* FDIS is ОПМС
* "ISO Guide" is "Руководства ИСО"
* "ISO/IEC Guide" is "Руководство ИСО/МЭК"

NOTE: More information can be found here: https://ru.delta-engineering.be/iso

All the following identifiers come from official ISO information, mostly from the website.
```
ИСО 10335
ИСО 1126
ИСО 124
ИСО 127
ИСО 1304
ИСО 1431-1
ИСО 1435
ИСО 15105
ИСО 18064
ИСО 2321
ИСО 289-1
ИСО 289-2
ИСО 289-3
ИСО 2930
ИСО 35
ИСО 37001
ИСО 3858
ИСО 4223-1
ИСО 4223-2
ИСО 45000
ИСО 4649
ИСО 4660
ИСО 472
ИСО 48
ИСО 506
ИСО 5435
ИСО 6107-6
ИСО 6107-6:1996
ИСО 6194-2
ИСО 8330
ИСО 8511
ИСО 8942
ИСО 9001:2015
ИСО 9026
ИСО/МЭК 76:2008
ИСО/МЭК ТО 19791:2008 # This is ISO/IEC
ИСО/ОПМС 26000:2010(R) # ОПМС is FDIS
ИСО/ОПМС 9000:2000
ИСО/ПМС 1956/2 # ПМС is DIS
ИСО/ТК
ИСО/ТО 8517 # This means ISO/TR 8517
ИСО/ТС 18625 # This means ISO/TS 18625
Руководства ИСО 2
Руководства ИСО 82
Руководство ИСО/МЭК 76
```

### PR DESCRIPTION
**Pull request by mico**
closes #8

### CODE REVIEW

File: lib/pubid/iso/parser.rb [hunk 1]
```diff
@@ -40,7 +42,10 @@ class Parser < Parslet::Parser
     #       #   "pas" / "r" / "tr" / "ts" / "tta"
     rule(:type) do
       (str("DATA") | str("ISP") | str("IWA") | str("R") | str("TTA") |
-        str("TS") | str("TR") | str("PAS") | str("Guide") | str("GUIDE")).as(:type)
+        str("TS") | str("TR") | str("PAS") | str("Guide") | str("GUIDE") |
+        # Russian types
+        str("ТС") | str("ТО")
```
Comment (ronaldtse) [inline lib/pubid/iso/parser.rb:47]: Are there Russian names for PAS, IWA, ISP, TTA and R (R stands for recommendation)?

File: lib/pubid/iso/transformer.rb [hunk 2]
```diff
@@ -28,8 +28,12 @@ class Transformer < Parslet::Transform
 
     rule(type: simple(:type)) do
       { type: case type
-              when "GUIDE"
+              when "GUIDE", "Руководство", "Руководства"
                 "Guide"
+              when "ТС"
```
Comment (ronaldtse) [inline lib/pubid/iso/transformer.rb:33]: Add a comment that this is for Russian.

File: lib/pubid/iso/transformer.rb [hunk 3]
```diff
@@ -28,8 +28,12 @@ class Transformer < Parslet::Transform
...
+                "TS"
+              when "ТО"
```
Comment (ronaldtse) [inline lib/pubid/iso/transformer.rb:35]: Add a comment that this is for Russian.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by mico** [comment]
@ronaldtse I believe I extracted everything I could find on https://ru.delta-engineering.be/iso

**Review by ronaldtse — APPROVED** [review]
Thank you @mico !

**Pull request merged**